File: src/week1/ji/Leet_171.java
class Solution {
	public int titleToNumber(String columnTitle) {
		int res = 0;
		for (int i = 0, j = columnTitle.length() - 1; i < columnTitle.length(); i++, j--) {
			res += (columnTitle.charAt(i) - 64) * (int) Math.pow(26, j);
		}

		return res;
	}
}

File: src/week1/ji/Leet_171.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        res = 0
        for i in range(len(columnTitle)):
            j = len(columnTitle) - i - 1
            res += (ord(columnTitle[i]) - 64) * (26 ** j)
        
        return res
